### ISSUE TITLE
[Bug]: Supabase: An error occurred while setting up the SSL connection

### ISSUE THREAD

**Issue by tschuehly** [coollabsio/coolify#3050]
### Description

Hello!

I've set up Supabase on a fresh coolify install and can successfully connect to the supabase dashboard. But I cannot connect to the Postgres DB. I've tried both with port 8000 and 5432. The error happens both with JDBC and in IntelliJ.

<img width="891" alt="image" src="https://github.com/user-attachments/assets/d63f4465-f2dc-4b64-878b-2ab9f09f149b">

### Minimal Reproduction (if possible, example repository)

Set up coolify, and supabase. Add a domain record and change it to the domain. Try to connect with a JDBC client.

### Version

v4.0.0-beta.323

### Cloud?

- [ ] Yes
- [X] No

**Comment by djsisson** [comment]
have you amended the compose file to expose the ports from the container to the host?

**Comment by tschuehly** [comment]
> have you amended the compose file to expose the ports from the container to the host?

Nope, isn't that happening via Kong? Then that's an easy fix, will try it out.

**Comment by djsisson** [comment]
you could also just use an ssh tunnel

**Comment by roshi112** [comment]
> > have you amended the compose file to expose the ports from the container to the host?
> 
> Nope, isn't that happening via Kong? Then that's an easy fix, will try it out.

hello can you please explain, i am facing same problem, the .well-known directory is not public thus giving 401 error when issuing the ssl.

kind regards.
